sdk/react-native/src/DVCProvider.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { DVCProvider as ReactDVCProvider } from '@devcycle/devcycle-react-sdk'
+import ReactNativeStore from './ReactNativeCacheStore'
+
+type configType = Parameters<typeof ReactDVCProvider>[0]
```

Comment:
this would be more accurately named "propsType"

Reply:
and conventionally would probably be `PropsType`

Reply:
yup! makes sense.. I was trying to make this only the `Parameters<typeof ReactDVCProvider>[0]['config']` but then didn't and forgot to rename the type back.. oops